woodwork/tests/accessor/test_accessor_utils.py

```diff
@@ -21,14 +21,17 @@
 
 def test_init_series_valid_conversion_specified_ltype(sample_series):
     series = init_series(sample_series, logical_type='categorical')
-    assert series is not sample_series
+    assert series is sample_series
```

Comment:
I think with the change to the dtype of `sample_series` we now never test converting a series dtype with Koalas like we did before. Maybe we should update this to convert sample series to have a different dtype at the start in this case, and leave the rest of the test the same?

I think this is one test where having sample_series not have a `category` dtype was useful.

Reply:
ahh, got it. I can change the dtype back to `'object'` for this and the other init_series tests at the start to keep the test the same as it was before, if you think that makes sense

Reply:
Yeah, I think that makes sense.

Reply:
okay, changed!